test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -176,4 +180,35 @@ private String executeAndRead(String command, Object... args)
             throw new RuntimeException(e);
         }
     }
+
+    public void captureLogs(String testName)
+    {
+        int count = 1;
+        String clusterLogsPath = ccmDir + "/" + DEFAULT_CLUSTER_NAME;
+        String folderName = testName;
+        String existingFolder = CASSANDRA_DIR + "/build/test/logs/validation/" + folderName;
+        ArchiveClusterLogs capture = new ArchiveClusterLogs();
+
+        if(capture.checkForFolder(existingFolder))
+        {
+            capture.zipExistingDirectory(existingFolder);
+        }
+
+        while (count <= nodeCount)
```

Comment:
Nope. You just happen to be instantiating a count variable prior to the loop, incrementing it at the end of the loop, and terminating the loop when you hit a certain count. It's perfectly suited for a for loop, and will look marginally cleaner.